:woman_farmer: Error finding gz-cmake3 package build regression

<!-- If you're not sure whether your problem is a bug, please ask a question at
https://robotics.stackexchange.com instead.-->

## Environment
Gz version: Harmonic
Affected builds: 
- https://build.osrfoundation.org/job/gz_sim-8-win
- https://build.osrfoundation.org/job/gz_rendering-8-win/
- https://build.osrfoundation.org/job/gz_launch-7-win
- https://build.osrfoundation.org/job/gz_gui-8-win
## Description
* Expected behavior: Builds to pass and find gz-cmake3 
* Actual behavior: Builds fail finding gz-cmake3 even though that is built from source from the job. 

## Steps to reproduce
<!-- Provide steps so we can try to reproduce this issue -->

1. Run a build of any of the jobs listed above
2. See it fail
## Output
```
CMake Error at C:/vcpkg/scripts/buildsystems/vcpkg.cmake:782 (_find_package):
  By not providing "Findgz-cmake3.cmake" in CMAKE_MODULE_PATH this project
  has asked CMake to find a package configuration file provided by
  "gz-cmake3", but CMake did not find one.

  Could not find a package configuration file provided by "gz-cmake3" with
  any of the following names:

    gz-cmake3Config.cmake
    gz-cmake3-config.cmake

  Add the installation prefix of "gz-cmake3" to CMAKE_PREFIX_PATH or set
  "gz-cmake3_DIR" to a directory containing one of the above files.  If
  "gz-cmake3" provides a separate development package or SDK, be sure it has
  been installed.
Call Stack (most recent call first):
  CMakeLists.txt:12 (find_package)
```

--- a/package.xml
+++ b/package.xml
@@ -10,7 +10,7 @@
 
   <buildtool_depend>cmake</buildtool_depend>
 
-  <build_depend>gz-cmake</build_depend>
+  <build_depend>gz-cmake3</build_depend>
 
   <depend>glut</depend>
   <depend>gz-common5</depend>

